### DISCUSSION TITLE
Help with site-2-site tunnel

### DISCUSSION THREAD

**Discussion by zloyBarsuk12** [strongswan/strongswan#1452] [Q&A]
Hello!!
I need to create a tunnel between one virtual machine with strongswan (on local network) and antoher on hosting. I want redirect some users on our network to this tunnel. Problem is how to config this virt machines? I need route-based vpn? Right machine on hosting need to redirect all traffic to internet. There is no local network on right machine. This is some kind of vpn channel for our network users.

Please help me)) Need some insights )

**Comment by tobiasbrunner** [comment]
I don't really get what you're asking. Maybe [this page](https://docs.strongswan.org/docs/5.9/howtos/forwarding.html) has some useful information for you?

**Comment by zloyBarsuk12** [comment]
![Screenshot_8](https://user-images.githubusercontent.com/120111427/206523354-64729c73-78fc-414a-a7cb-8e441265f02a.jpg)

**Reply by zloyBarsuk12** [reply]
What the best options to create this type of tunnel?

**Reply by tobiasbrunner** [reply]
What do you mean? Seems like a simple enough scenario.

**Comment by Thermi** [comment]
You can do this either with policy based tunnel or with route based one. I propose you use a route based one and also use PBR on the virtual machines to only route traffic of the users through the tunnel.

**Reply by zloyBarsuk12** [reply]
Thank you! Do you have some config examples for this scenario?

**Comment by misch42** [comment]
https://docs.strongswan.org/docs/5.9/config/IKEv2.html#_site_to_site

and

https://docs.strongswan.org/docs/5.9/features/routeBasedVpn.html

but a policy based VPN, simple VPN as described in the examples above should also do if you do the policing on the nexus.

**Comment by misch42** [comment]
Also another thought: Why don't you set up the VPN from your Nexus. You would not need the strongswan machine.
